go/vt/schemadiff/table_test.go

```diff
@@ -1188,11 +1217,7 @@ func TestValidate(t *testing.T) {
 			applied, err := from.Apply(a)
 			if ts.expectErr != nil {
 				assert.Error(t, err)
-				if errors.Is(err, ts.expectErr) {
-					assert.ErrorIsf(t, err, ts.expectErr, "error mismatch. expected: %v, got: %v", ts.expectErr, err)
-				} else {
-					assert.EqualError(t, err, ts.expectErr.Error())
-				}
```

Comment:
This was a left over that accidentally stayed from when I was refactoring this to the structured error types. Now that everything converted, this is no longer needed.